tools/mock_ehr/app.py

```
@@ -0,0 +1,162 @@
+                    status_code=502,
+                    detail=f"Could not reach Shadi: {exc}",
+                ) from exc
+
+        detail: dict[str, Any] = {
+            "shadi_status_code": r.status_code,
+            "encounter_id": eid,
+            "patient_id": body.patient_id,
+        }
+        try:
+            detail["shadi_body"] = r.json()
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)